Tests.Integration.Presentation.Web/SystemUsage/V2/ItSystemUsageApiV2Test.cs

```diff
@@ -506,6 +506,29 @@ public async Task Can_Get_Specific_ItSystemUsage_Permissions(OrganizationRole us
             Assert.Equivalent(expected, permissionsResponseDto);
         }
 
+        [Theory]
+        [InlineData(OrganizationRole.LocalAdmin, true)]
```

Comment:
global admin should also be able to do it :-)
Remember, global admin is a user property and not a role which can be assigned